etl/requirements.txt

```diff
@@ -1 +1,2 @@
 pyodbc==4.0.34
+pyproj==3.0.1
```

Comment:
Just a couple of quick points here -- it's worth a comment to explain why this isn't the latest version, and also I've started checking the "pip freeze" output instead of the top level dependencies (with the rationale that this makes dependency checking more robust because you know exactly what you have installed)